.taskcluster.yml

```diff
@@ -67,11 +68,134 @@ tasks:
           && git fetch origin --tags
           && git config advice.detachedHead false
           && git checkout {{ event.version }}
-          && ./gradlew --no-daemon clean test detektCheck ktlint assembleRelease docs lint
+          && ./gradlew --no-daemon clean test detektCheck ktlint assembleRelease docs lint uploadArchives zipMavenArtifacts
           && python automation/taskcluster/release/fetch-bintray-api-key.py
           && ./gradlew bintrayUpload --debug
       features:
         taskclusterProxy: true
+
+      artifacts:
+        public/build/lib.dataprotect.maven.zip:
+          path: /build/android-components/components/lib/dataprotect/build/target.maven.zip
+          expires: "{{ '1 year' | $fromNow }}"
```

Comment:
Is 1 year the default? If so, can we remove the config? Technically it doesn't matter that the artifacts stay around once they're in our Maven repo, right?

Reply:
You're right it is 1 year. I remember it used to be less, when we worked on Firefox Focus, 6 months ago. The documentation says the value may change again https://github.com/taskcluster/taskcluster-queue/blob/3b2960adf20ccb960a474fd68a526b4c389868b4/schemas/v1/task.yml#L132 

I'd be in favor of keeping them for a year, in case we need to audit the artifacts present on Maven. That's what we do on Fennec. 

Therefore, I think we should keep 1 year explicitly defined.